docs/topics/hacking/debugging.rst

```diff
@@ -46,5 +46,38 @@ All being well it should look like this:
 .. image:: /screenshots/docker-ipdb.png
 
 
+Using the Django Debug Toolbar
+------------------------------
+
+The `Django Debug Toolbar`_ is very powerful and useful when viewing pages from
+the website, to check the view used, its parameters, the SQL queries, the
+templates rendered and their context...
+
+To enable it, create a `local_settings.py` file with the following content::
+
+    from settings import *  # noqa
+
+    INSTALLED_APPS += (
+        'debug_toolbar',
+    )
+    DEBUG_TOOLBAR_PATCH_SETTINGS = False  # Prevent DDT from patching the settings.
+
+    INTERNAL_IPS = ('127.0.0.1', '192.168.99.1')
+    MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
+
+The `192.168.99.1` IP address should be the IP address of your own computer,
```

Comment:
Technically this is the ip of the host machine on the host-only network provided by virtualbox when used via docker-machine.